apps/slides/models.py

```diff
@@ -17,54 +17,53 @@ class Slide(models.Model):
 	    (28, '28 px'),
 	)
 	created		  = models.DateTimeField(default=datetime.now())
-	text		  = models.TextField("Enter Event Info (as it should appear on marquee):", default="Text to display")
+	text		  = models.TextField("Enter Event Info (as it should appear on marquee):", default="Event\nDate\nTime\nLocation")
 	font		  = models.CharField(default='Gotham', max_length='255', choices=fonts_tuple())
 	font_size     = models.IntegerField("Font Size", default=16, choices=FONTSIZES)
 	color_bg	  = models.CharField("Background Color", max_length="6", default="000000")
 	color_font	  = models.CharField("Font Color", max_length="6", default="EBB700")
 	display_start = models.DateField()
 	display_end	  = models.DateField()
-	event_start	  = models.DateTimeField(null=True)
-	event_end	  = models.DateTimeField(null=True)
-	location	  = models.CharField("Event Location", max_length='255', blank=True)
 	org			  = models.CharField("Name of UCF Organization / Unit", max_length=255)
```

Comment:
We'll need to get migrations set up and included in this PR before it can be merged in.

Reply:
Migrations done by Jim